.gitignore

```diff
@@ -59,4 +59,5 @@ node_modules/
 # Any YAML in config
 **/providers/*.yaml
 
-.vagrant
+/.vagrant
```

Comment:
I would but I only wanted to ignore those 2 directories in the root.. does that mean i need pre and trailing slashes?